File: execs/python/11.py
import math

def isPrime(num):
	if num==1:
		prime=False
	elif num==2:
		prime=True
	else:
		prime=True
		for check in range(2,int(math.sqrt(num))+1):
			if num%check==0:
				prime=False
				break
	return prime

number=int(input("Enter a number: "))
if(isPrime(number)):
	print number,"is prime"
else:
	print number," is not prime"

File: execs/php/11.php
<?php


echo "<form action=\"\" method=\"post\">
Enter a number: <input type=\"text\" name=\"num\"><br>
<input type=\"submit\" value=\"Choose\">";




function isPrime($num)
{
	$prime=1;
	if($num==1)
	{	
		$prime=0;
	}	
	elseif($num==2)
	{
		$prime=1;
	}
	else
	{
		for($i=2;$i<sqrt($num)+1;$i+=1)
		{
			if($num%$i==0)
			{	
				$prime=0;
				break;
			}			
		}
	}
	return $prime;
}
if(isset($_POST['num']))
{
$num1 = $_POST['num'];

if(isPrime($num1))
	echo "</br>$num1 is prime.";
else
	echo "</br>$num1 isn't prime.";
}

?>

